src/NativeShell/src/cordova/NativeShell.java

```diff
@@ -0,0 +1,188 @@
+package org.jellyfin.cordova;
+
+import android.os.Build;
+import android.provider.Settings;
+import android.content.Intent;
+import android.support.v7.appcompat.BuildConfig;
+import android.view.WindowManager;
+
+import org.apache.cordova.CallbackContext;
+import org.apache.cordova.CordovaPlugin;
+import org.apache.cordova.CordovaWebView;
+import org.apache.cordova.PluginResult;
+import org.jellyfin.mobile.R;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import com.mb.android.media.RemotePlayerService;
+
+import mediabrowser.apiinteraction.android.mediabrowser.Constants;
+
+import static android.view.View.SYSTEM_UI_FLAG_FULLSCREEN;
+import static android.view.View.SYSTEM_UI_FLAG_HIDE_NAVIGATION;
+import static android.view.View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY;
+import static android.view.View.SYSTEM_UI_FLAG_VISIBLE;
+
+public class NativeShell extends CordovaPlugin {
+    public CallbackContext callbackContext;
+    public JSONArray args;
+
+    public static CordovaWebView cordovaWebView;
+
+    @Override
+    public void pluginInitialize() {
+        cordova.getActivity().runOnUiThread(new Runnable() {
+            @Override
+            public void run() {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                    cordova.getActivity().getWindow().setStatusBarColor(cordova.getActivity().getResources().getColor(R.color.gray));
+                    cordova.getActivity().getWindow().setNavigationBarColor(cordova.getActivity().getResources().getColor(R.color.gray));
+                }
+            }
+        });
+    }
+
+    @Override
+    public boolean execute(String action, JSONArray args, final CallbackContext callbackContext) {
+        this.callbackContext = callbackContext;
+        this.args = args;
```

Comment:
ain't it thread-unsafe if there're several parallel requests?
seeing this is used in one or two methods I suggest making `args` the argument for those methods instead of passing it via `this`.